br/pkg/task/restore.go

```diff
@@ -1358,6 +1365,24 @@ func Exhaust(ec <-chan error) []error {
 	}
 }
 
+func checkTableExistence(ctx context.Context, mgr *conn.Mgr, tables []*metautil.Table, glue glue.Glue) error {
+	if glue.GetType() != "ExecGlue" {
```

Comment:
Perhaps make things like `"ExecGlue"` constants. 

Also would you add some comments about why we need to skip check table existence when executing type isn't `"ExecGlue"`? Why this check cannot be done at the caller side?